sonar-db/src/main/resources/org/sonar/db/permission/PermissionMapper.xml

```diff
@@ -184,7 +183,7 @@
     select 0 as group_id, 'Anyone' as name, gr.role as permission, gr.resource_id as resource_id, gr.id as id
     from group_roles gr
     <where>
```

Comment:
I didn't touch the requests of this mapper. My goal is not to refactor everything !